### ISSUE TITLE
The new progress bar is very uninformative

### ISSUE THREAD

**Issue by moonmehta** [artemanufrij/imageburner#40]
The new progress bar doesn't give any idea how much of the job is done and doesn't give the user an expected waiting time. Please give a % indicator or at least a bar indicator.

![screenshot from 2018-04-01 15-51-45](https://user-images.githubusercontent.com/22585732/38172176-f7aa11c2-35c4-11e8-860f-5ed7f242a12e.png)

**Comment by artemanufrij** [comment]
the problem is `dd` doesn't return exact progress state. Very often it has following behaviour: 0% -> [instant] ->  94% -> [wait,wait,wait] -> 100%.

I have plans to switch to `udisk` #3. I guess it would solve the issue with progress value.

**Comment by leoherzog** [comment]
Not to mention, no indication or notification when it's done. If you start a process and come back to the window, it's hard to know if it actually ran without checking the disk manually.

**Comment by FDiskas** [comment]
Yes, first then I tried this app and so this wait wait wait... I killed that app - thought it stuck or something

**Comment by Julioevm** [comment]
I agree we should at least get a confirmation the process has finished successfully. It's very uninformative this way.

**Comment by nandgator** [comment]
The only indication I get is constant flickering of the USB. So pretty much anxiety hangs in the air if there is no proper progress bar. Fortunately there's a notification if the task is completed in eOS :)
